index.html

```diff
@@ -21,17 +21,19 @@
   <body>
     <header class="container">
```

Comment:
- [ ] The navigation should use the `<nav>` tag
 screenshot from requirements 👇🏾 
![Screen Shot 2021-03-01 at 7 23 59 PM](https://user-images.githubusercontent.com/48804258/109541094-ad525e80-7ac3-11eb-8ba2-d055a0619d74.png)